src/js/helpers/checkboxTreeHelper.js

```diff
@@ -57,7 +57,7 @@ export const removePlaceholderString = (str) => {
 };
 
 export const getAllDescendants = (node) => {
-    if (!node.children) return [node.value];
+    if (!node.children || node?.children?.length === 0) return [node.value];
```

Comment:
do we need this `node?.children?.length` couldn't it be `node?.children.length` since children is always an array or has that changed?

Reply:
children is always an array. that hasn't changed. but I dont think the chain operator makes children an array if it's undefined, so `undefined/value-of-chain-operator-when-prop-does-not-exist.length` probably won't work w/o another chain operator.

Reply:
node.children though could be absent from the object altogether.

Reply:
yeah, I think I have been reading it wrong from when I started using it. Thanks